### DISCUSSION TITLE
wallpaper

### DISCUSSION THREAD

**Discussion by Vashirr01** [mylinuxforwork/dotfiles#488] [General]
after i installed the seqouia theme, there is no wallpaper seen(just a black screen). waypaper works fine and the terminal changes colour, but the wallpaper is not rendered.

**Comment by mylinuxforwork** [comment]
Can you press super+shift+w to chane the wallpaper and try again?

**Reply by Vashirr01** [reply]
tried, still the same

**Comment by Veicm** [comment]
It could help to open the ml4w welcome app and click on the three dots then click on a button wich contains "sddm" or something similar. If you read it it should be obvious wich button will give you the solution.
